src/compiler/crystal/command.cr

```diff
@@ -298,6 +308,12 @@ class Crystal::Command
         opts.on("--no-debug", "Skip any symbolic debug info") do
           compiler.debug = Crystal::Debug::None
         end
+
+        {% if flag?(:darwin) %}
+          opts.on("--no-dsymutil", "Don't invoke dsymutil") do
```

Comment:
I don't find this description helpful.
Also, why not bunch this together with `--no-debug`

Reply:
Because `--no-debug` makes it impossible to debug with `gdb` or `lldb`. But even if passing `--no-dsymutil` you can debug with `gdb` or `lldb`.

But yes, I think we can just remove this option, I don't mind having this `dwarf` file generated every time.